packages/pluggableWidgets/notifications-native/src/Notifications.tsx

```diff
@@ -49,6 +57,17 @@ export function Notifications(props: NotificationsProps<undefined>): null {
             if (actions.length === 0) {
                 return;
             }
+
+            if (id !== undefined) {
+                if (knownIds.has(id)) {
+                    console.debug(`Not executing handleNotification for ${id}, has been executed before`);
```

Comment:
why should we keep this logs?

Reply:
If it doesn't add value to the customer, it should be removed.

Reply:
Thanks, Removed 👍